src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/electronmicroscopy/ElectronMicroscopyParameters.groovy

```diff
@@ -1,12 +1,35 @@
 package life.qbic.datamodel.dtos.imaging.parameters.electronmicroscopy
 
+import life.qbic.datamodel.dtos.imaging.properties.Voltage
+
 /**
- * <class short description - 1 Line!>
+ * Describes the properties of electron microscopy
+ *
+ * This class serves as DTO and contains information about
+ * general electron microscope parameters during an image analysis.
  *
- * <More detailed description - When to use, what it solves, etc.>
+ * This class shall be extended, if you need to add more technology-specific
+ * properties.
  *
  * @author Sven Fillinger
- * @since <versiontag>
+ * @since 1.0
  */
 class ElectronMicroscopyParameters {
+
+    /**
+     * Voltage used to accelerate electrons.
+     * Applied between cathode and electrode in the acceleration tube
+     */
+    private final Voltage acceleratingVoltage
+
+    /**
+     * Shift of focus of the objective lens.
+     * It is used to observe 'Fresnel fringes', for taking lattice image or structure images.
+     */
+    private final Float targetDefocus
+
+    /**
+     * The accumulated amount of electrons transmitting the materials
+     */
+    private final Float cumulativeElectronDose
```

Comment:
Do we now sth about the common units?

Reply:
I tried to be more specific, but i was unsure since the documentation on such properties is not so good / seems to vary between different microscopes